docs/samples/Microsoft.ML.Samples/DatasetCreator.cs

```diff
@@ -44,14 +44,14 @@ public static (string trainPath, string testPath) CreateRegressionDataset()
                 File.WriteAllText(trainDataPath, csvTrain.ToString());
             else
             {
-                new Exception("Train dataset file already exists");
+                throw new Exception("Train dataset file already exists");
```

Comment:
>throw new Exception("Train dataset file already exists"); [](start = 16, length = 57)

This might be a little annoying when calling multiple times this function. I wonder if it is better to overwrite the file in case it is already present?

Reply:
how about
```
bool TryCreateRegressionDataset(out string trainDatasetPath, out string testDatasetPath)
```

---
In reply to: [223520000](https://github.com/dotnet/machinelearning/pull/1189#discussion_r223520000) [](ancestors = 223520000)

Reply:
Actually will delete this file, now that we don't want to do a nget for the datasets, and just download from github. 

---
In reply to: [223520671](https://github.com/dotnet/machinelearning/pull/1189#discussion_r223520671) [](ancestors = 223520671,223520000)